src/extension/constants.ts

```diff
@@ -1,9 +1,13 @@
 export const REQUEST_TAB_ID = "request-tab-id";
-export const DEVTOOLS_INITIALIZED = "devtools-initialized";
-export const CREATE_DEVTOOLS_PANEL = "create-devtools-panel";
+export const CLIENT_NOT_FOUND = "client-not-found";
```

Comment:
I did not choose to prefix these quite yet. There is more I'd like to investigate with our communication between the various areas of our extension, but I'd like to do that all at once instead of combining that in this PR.